updating/understanding-upgrade-channels-release.adoc

```diff
@@ -33,3 +33,6 @@ ifdef::openshift-origin[]
 endif::openshift-origin[]
 
 include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+
+.Additional resources
+* For more information, see xref:../updating/updating-cluster-cli.adoc#update-conditional-upgrade-pathupdating-cluster-cli[Updating to a conditional upgrade path]
```

Comment:
The title of the linked page is "Updating along a conditional upgrade path", but the ID is "Updating to a conditional upgrade path." Not sure if it needs changed, just FYI!